### ISSUE TITLE
508-defect-4 [SCREENREADER]: Excessive sr-only text can create a verbose and annoying experience to screen reader users OR an unequal experience for sighted users 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#26352]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I don't want to listen to repetitive or overly verbose content that sighted people don't have to scan through so that I can navigate through an experience faster.

## Details

Having `sr-only` (screen reader only) text can negatively impact a screen reader user's experience when it's overused in inappropriate contexts. It can also create unequal experiences for sighted users who may be missing out from important details that only screen reader users can hear.

<del> VA.gov Search Results </del>

<del> Screen reader users are provided with an additional instruction that they can use phrases, but sighted users are not.
If this is important, delete the `sr-only` text and add it to the visible label so all users have access to equal instructions. </del>

<del> `hr` tags are announced </del>
<del> Prevent `hr` tags from being announced to screen reader users by using `aria-hidden="true"`.</del>

## Resources and Support Search Results

https://user-images.githubusercontent.com/14154792/122806228-14289d00-d298-11eb-935a-ee833cf34079.mov

### Verbose search field
Remove unneeded `sr-only` text like "website section to search," "article type," and "Search within". Consider rewriting the content within the radio button fieldset to be less verbose to screen reader users.
<img width="1176" alt="Screen Shot 2021-06-21 at 1 36 41 PM" src="https://user-images.githubusercontent.com/14154792/122806297-2a365d80-d298-11eb-87ea-797129d58bfe.png">

<img width="1044" alt="Screen Shot 2021-06-21 at 1 36 16 PM" src="https://user-images.githubusercontent.com/14154792/122940603-9cfc1300-d342-11eb-821e-7b3bb32cf9ed.png">

**Comment by denisecoveyduc** [comment]
@srsuddath has addressed the two global search components of this ticket (Screen reader users are provided with an additional instruction that they can use phrases, but sighted users are not) and (hr tags are announced)

@brianalloyd @kelsonic  Could you confirm, if the resources and support fixes have been done or are needed? Thanks!

**Comment by brianalloyd** [comment]
@denisecoveyduc I don't believe so, I'm going to create a new ticket for our team to address this action and connect it to this ticket for reference.

**Comment by joshkimux** [comment]
Also @denisecoveyduc , thank you for the update! Appreciate @srsuddath 's solid a11y work as always. I'll strike those out from this ticket!

**Comment by brianalloyd** [comment]
Will do @joshkimux appreciate the follow up.

**Comment by joshkimux** [comment]
Since we have new tickets written up for the final item here, I'll close this out!
